### PR TITLE
Uncensorability

### PR DESCRIPTION
**Pull request by dionyziz**
Depends #7

### CODE REVIEW

File: analysis.tex [hunk 1]
```diff
@@ -61,31 +79,202 @@ \section{Analysis}
...
+  by $\textsf{Gen}$ with some probability $p$.
+  Consider the existential forgery adversary $\mathcal{A}$ which works as
+  follows. It receives $pk$ as its input from the challenger, but ignores it
+  and generates a new key pair $(pk', sk') \gets \textsf{Gen}(1^\kappa)$.
+  Since the
+  two invocations of $\textsf{Gen}$ are independent,
+  $\Pr[pk = pk'] \geq \Pr[pk = \widehat{pk} \land pk' = \widehat{pk}]
+  = \Pr[pk = \widehat{pk}]\Pr[pk = \widehat{pk'}] = p^2$. The adversary checks
+  whether $pk = pk'$. If not, it aborts. Otherwise, it uses $sk'$ to sign the
+  message $m = \epsilon$ and returns the forgery $\sigma = \textsf{Sig}(sk, m)$.
```
Comment (gtklocker) [inline analysis.tex:141]: What is \epsilon here?

**Review by gtklocker — CHANGES_REQUESTED** [review]
Very nice.

Please make sure the formatting for negl is uniform throughout the paper (it's not sf on the rest of the paper). Same for the formatting of the challenger names (textsc vs textsf).

File: analysis.tex [hunk 2]
```diff
@@ -61,31 +79,202 @@ \section{Analysis}
...
+  Let $\mathcal{T}$ be an unpredictable distribution ensemble and $H$ be a
+  Random Oracle.
+  The distribution $X = \{t \gets \mathcal{T}; H(t)\}$ is indistinguishable from
+  the uniform distribution $\uniformk$.
+\end{lemma}
+\begin{proof}
+  Let $\mathcal{A}$ be an arbitrary polynomial distinguisher between
+  $X$ and $\uniform(\{0, 1\}^\kappa)$.
+  We construct an adversary $\mathcal{A}^*$
+  against $\textsf{predict}_{\mathcal{T}}$.
```
Comment (gtklocker) [inline analysis.tex]: should be sc

**Review by gtklocker — CHANGES_REQUESTED** [review]

File: analysis.tex (hunk 1 above)
Comment (dionyziz) [reply analysis.tex:141]: The empty string.

Comment (dionyziz) [reply analysis.tex:141]: I now defined it in the notation section.

**Review by gtklocker — APPROVED** [review]

**Pull request merged**